.github/workflows/reusable-unit.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           - react-native
           - react-core
           - react-storage
+          - react-liveness
```

Comment:
Thinking you also need to build any dependencies, not just add it to the matrix? DO you need to build react package perhaps? See: https://github.com/aws-amplify/amplify-ui/blob/main/.github/workflows/reusable-unit.yml#L83

Reply:
yep, good callout